gym/envs/box2d/bipedal_walker.py

```diff
@@ -608,8 +608,8 @@ def step(self, action: np.ndarray):
     def render(self):
         if self.render_mode is None:
             gym.logger.warn(
```

Comment:
Should this be a warning or an error? I think it should be an error as otherwise users will ignore it and they shouldn't be ignoring it. 
Also I think, we should have the render_mode defaulting to human as that was the previous default used

Reply:
Since now in this if branch there is a return, calling render with `render_mode=None` simply results in doing nothing. Thus, I would not throw an error. I can see a user training an agent for a long time and render in the end resulting in an error while it can be just a warning since nothing bad happens. 

Regarding going back to `render_mode='human'` as default, if we want to do that, we would need again a deprecation period. Personally, I don't see the advantage of having `human` as default. I think rendering should be something that you explicitly turn on.

Reply:
Sorry, about "human" I mean in the message instead of "rgb_array"
`gym.make("CartPole-v1", render_mode="human")`

Reply:
Ah, got it!
I didn't do that because with `render_mode="human"` you don't have to call `render`. It would be a bit confusing to suggest `human` as a warning because you called `render`. 
What do you think?

[Btw, right now, calling render with human works anyway without warning, it is just useless. We had a warning during the deprecation period and it is stated in doc.]

Reply:
Agreed, but people most likely to get an issue is those calling render without an parameters which use to default to human. If users wanted a different render mode then they would have needed to enter a mode parameter and get an error prior to this warning. 

However, either mode would work